scripts/xyz_grid.py

```diff
@@ -484,6 +484,11 @@ def process_axis(opt, vals):
         z_opt = self.current_axis_options[z_type]
         zs = process_axis(z_opt, z_values)
 
+        # this could be moved to common code, but unlikely to be ever triggered anywhere else
+        Image.MAX_IMAGE_PIXELS = opts.img_max_size_mp * 1.1 # allow 10% overhead for margins and legend

```

Comment:
This field set should not be here.
... A, in the first place at all, but also B the unit is different (pixels vs megapixels)

Reply:
The goal is to avoid Pillow decompression bomb runtime errors that nobody understood and was creating issues and replace them with easily understandable and configurable option. But latest update of the PR created a typo, fixed.